src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/custom_op_insert_convert_i64.cpp

```diff
@@ -81,42 +106,44 @@ class CustomOpConvertI64CPUTest : public testing::WithParamInterface<CustomOpI64
                                   public CPUTestsBase {
 public:
     static std::string getTestCaseName(const testing::TestParamInfo<CustomOpI64CPUTestParams>& obj) {
-        ElementType inType;
-        InputShape inputShape;
-        std::tie(inType, inputShape) = obj.param;
-
         std::ostringstream result;
-        result << "IS=" << inputShape << "_";
-        result << "Prc=" << inType;
+
+        result << "InElType=" << std::get<0>(obj.param);
+        result << "_IS="      << std::get<1>(obj.param);
+
         return result.str();
     }
 
 protected:
     void SetUp() override {
-        targetDevice = ov::test::utils::DEVICE_CPU;
-
-        ElementType inType;
-        InputShape inputShape;
-        std::tie(inType, inputShape) = this->GetParam();
-
-        init_input_shapes({inputShape});
-        ov::ParameterVector inputParams;
-        ov::OutputVector paramsOuts;
-        for (auto&& shape : inputDynamicShapes) {
-            auto param = std::make_shared<ov::op::v0::Parameter>(inType, shape);
-            inputParams.push_back(param);
-            paramsOuts.push_back(param);
-        }
-        auto customOp = std::make_shared<CustomOpI64>(paramsOuts);
+        targetDevice = test::utils::DEVICE_CPU;
+
+        const auto& params = this->GetParam();
+        const auto& in_el_type = std::get<0>(params);
+        const auto& in_shape = std::get<1>(params);
+
+        init_input_shapes({in_shape});
+        auto param = std::make_shared<op::v0::Parameter>(in_el_type, inputDynamicShapes[0]);
+
+        auto custom_op_0 = std::make_shared<CustomOpI64>(param);
+        auto i32_op_0 = std::make_shared<op::v1::LogicalNot>(custom_op_0->output(0));
+        auto i32_op_1 = std::make_shared<op::v1::LogicalNot>(param);
+        auto custom_op_1 = std::make_shared<CustomOpI64>(i32_op_1);
 
-        ov::ResultVector results{std::make_shared<ov::op::v0::Result>(customOp)};
-        function = std::make_shared<ov::Model>(results, inputParams, "customOpTest");
+        OutputVector results{
+                i32_op_0->output(0),
+                custom_op_0->output(1),
+                custom_op_1->output(0),
+                custom_op_1->output(1)
+            };
```

Comment:
```suggestion
        OutputVector results{
            i32_op_0->output(0),
            custom_op_0->output(1),
            custom_op_1->output(0),
            custom_op_1->output(1)
        };
```

Reply:
I believe, it should be checked by Code Style task.